projects/storefrontlib/src/cms-components/product/product-tabs/product-details-tab/product-details-tab.component.html

```diff
@@ -1,3 +1,20 @@
 <ng-container *ngIf="product$ | async as product">
-  <div class="container" [innerHTML]="product?.description"></div>
+  <div class="container">
+    <div
+      *ngIf="!showMore; else fullContent"
+      [innerHTML]="product?.description.substring(0, 400)"
```

Comment:
it's better to create a const for this `400`.

Reply:
done